### ISSUE TITLE
Drag&drop seems erratic (sometimes works, sometimes not)

### ISSUE THREAD

**Issue by RespeQt** [RespeQt/RespeQt#7]
Several users have experienced issues while drag&droping files from their desktop to drive slots.

**Comment by AlecVoid** [comment]
Any progress on fixing this bug? Its been here for ages and its the same bug for all RespeQt/AspeQt versions (under Windows 7/10 anyways).

**Comment by AlecVoid** [comment]
I posted the same bug on the AspeQt git. Good news... while writing a follow up I found out how to reproduce this "erratic" error 100% of the time.

In the following video I use AspeQt, but its exactly the same behaviour with RespeQt:
https://user-images.githubusercontent.com/11749584/151678530-752c90e0-0216-4bcd-9427-47f97bd82c9f.mp4

As you hopefully can see: if I move the dragged file into the AspeQt window while exactly over a drive slot - all works well as long as I keep the dragged file in the AspeQt window.
If I enter the AspeQt window over a non-drive slot element the "forbidden icon" appears and it stays as long as I am in the AspeQt window.

Should be easy to fix. ;)

**Comment by AlecVoid** [comment]
.. one year later .. 

Is there really no one who could fix that bug? Is this project dead?

**Comment by josch1710** [comment]
The issue seems to be that the drive strips are the drop target, not the whole RespeQt window.
Which makes sense, because you want to connect an image to an specific device number.
Would adding an image to the next free slot when dropping somewhere else, be OK?

Edit: It seems that's exactly the intended fallback behaviour. I will try investigate the whole drop business further.

**Comment by AlecVoid** [comment]
Hey Josch,

it would be an ok fallback, but the problem is not the drive strips - the problem is that you can not get a valid drop target after(!) your dragging touched anything other(!) than the drive strips (i.e. the window space around the drive strips).

Regards,
AlecVoid
(AlexVG from Abbuc)

**Comment by josch1710** [comment]
Yes, I saw that after my comment. I need some time to investigate the issue, but I think, it's solvable.

**Comment by josch1710** [comment]
Ok. I fixed this.

**Comment by josch1710** [comment]
The Drap and Drop icon should change, when not inside a slot widget.
